### DISCUSSION TITLE
Finer steps when changing distance spacing with hotkeys

### DISCUSSION THREAD

**Discussion by zhevv** [ppy/osu#23864] [Ideas] [answered]
When using the hotkeys to change distance spacing, it increases/decreases in steps of **0.1**. It would be great to also have the option to change it with hotkeys in steps of **0.01** like in stable. 

With that, I'm suggesting to make it consistent with how the distance spacing slider in the left toolbox handles user input. By default, just using the hotkeys will increase/decrease in steps of **0.01**, but pressing hotkey + shift makes it go in steps of **0.1**.

**Comment by peppy** [comment]
Can you explain how to adjust by `0.01` in stable? I can't seem to make it work. Holding shift turns on grid snap.

**Reply by zhevv** [reply]
`shift + alt + scroll` changes it by `0.01`. Only doing `alt + scroll` changes it by `0.1`.

**Reply by peppy** [reply, marked as answer]
The worst part of this is that as you touch on, this seems to be back-to-front with how shift works everywhere else (holding shift *increases* the adjustment amount).

So I think I agree with your proposal.

**Reply by peppy** [reply]
WIP branch for this here: https://github.com/ppy/osu/compare/master...peppy:osu:distance-snap-precision-adjust?expand=1

TODO:

- Migration for framework bevaviour change (default keys written to user's config for beat divisor change need to be updated)
- Need to figure how to actually make the binding work when holding `Shift` (currently `GlobalActionContainer` is matching on `Modifiers` which doesn't allow this)
